app/models/user.rb

```diff
@@ -3,4 +3,5 @@ class User < ApplicationRecord
   has_many :posts, dependent: :destroy
   has_many :likes, dependent: :nullify
   validates :name, presence: true
+    scope :last_three_posts, ->(user) { Post.where(author: user).last(3) }
```

Comment:
- [ ] According to the guidelines you should have a method that returns 3 most recent posts. You can checkout the guide [Scope vs Class Methods](https://www.square63.com/scope-vs-class-methods/)

```ruby
def recent_post
  posts.order('created_at Desc').limit(3)
end
```